src/components/Checkbox/Checkbox.tsx

```diff
@@ -138,7 +138,7 @@ const Checkbox = React.forwardRef<HTMLInputElement, CheckboxProps>(
           {showLabel && labelText}
         </ChakraCheckbox>
         {footnote && showHelperInvalidText && (
-          <Box __css={styles.helper}>
+          <Box __css={styles.helper} aria-disabled={isDisabled}>
```

Comment:
This isn't necessary and gets removed in #925 .

Reply:
@EdwinGuzman I saw that this had been added to `Radio`, so I added it here as well.  However, with that, I noticed that some of the styles were not rendering when this was not applied.  I will remove here and they reassess when the other PR is merged.